src/components/portrait/portrait-initials.component.js

```diff
@@ -0,0 +1,110 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Browser from '../../utils/helpers/browser';
+
+class PortraitInitials extends React.Component {
+  static propTypes = {
```

Comment:
Can we move it to the bottom of the component and extract from static?
```
PortraitInitials.propTypes = {}
```

Reply:
See https://github.com/Sage/carbon/pull/2132#discussion_r297175982

Reply:
Updated in bfe2072 :thumbsup: